sky/skylet/events.py

```diff
@@ -136,13 +137,11 @@ def _stop_cluster(self, autostop_config):
         else:
             raise NotImplementedError
 
-    def _replace_yaml_for_stopping(self, yaml_path: str, terminate: bool):
+    def _replace_yaml_for_stopping(self, yaml_path: str, down: bool):
         with open(yaml_path, 'r') as f:
             yaml_str = f.read()
-        # Update the number of workers to 0.
-        yaml_str = self._NUM_WORKER_PATTERN.sub(r'\g<1>_workers: 0', yaml_str)
```

Comment:
Why is this not needed?

Reply:
1. Since the upscaling speed is set to `0`, after `ray down -y --workers-only`, there will be no more new workers launched by ray. Changing the workers to `0` is not needed.
2. If we set the worker to `0`, the `ray up -y --restart-only` will stop the workers no matter what is the value for `cache_stopped_nodes`, which causes resource leakage during `auto-down`.